django_react_proj/users-fe/src/components/SummingTime/SummingTime.js

```diff
@@ -0,0 +1,42 @@
+import React, { useEffect, useState } from 'react';
+import axios from 'axios';
+import { API_URL } from '../../constants';
+import styles from './SummingTime.module.css';
+
+const Subtract = () => {
+  const [users, setUsers] = useState();
+  let timevalue = 0;
+  const getUsers = () => {
+    axios.get(API_URL).then((res) => setUsers(res.data));
```

Comment:
Such request should also have some error handling in case of non-optimistic scenario. Use `.catch()` /  `try / catch`.

Maybe create separate task for error handling in overall? Like toasts / popups / error messages (not `console.log`)?